installTerragrunt/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "install-terragrunt",
-  "version": "0.18.3",
+  "version": "0.19.0",
```

Current version is `0.28.16`
```suggestion
  "version": "0.28.16",
```